rs/replicated_state/src/canister_state/queues/tests.rs

```diff
@@ -2223,7 +2543,7 @@ fn has_expired_deadlines_reports_correctly() {
 
     let time1 = Time::from_secs_since_unix_epoch(1).unwrap();
     canister_queues
-        .push_output_request(Arc::new(RequestBuilder::default().build()), time1)
+        .push_output_request(Arc::new(RequestBuilder::default().build()), time0)
```

Comment:
Do we also want a test that tests the same for best effort messages?

Reply:
Also enqueued a best-effort response (does not expire) and request (does expire) into the input queue (so we also test that).